### ISSUE TITLE
Error in api2html.js when building Wekan snap

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#2286]
https://launchpadlibrarian.net/416460186/buildlog_snap_ubuntu_xenial_amd64_wekan-devel_BUILDING.txt.gz

```
added 2 packages from 1 contributor in 0.826s
[25/Mar/2019:15:37:46 +0000] "CONNECT registry.npmjs.org:443 HTTP/1.1" 200 9459 "-" "-"
[25/Mar/2019:15:37:46 +0000] "CONNECT registry.npmjs.org:443 HTTP/1.1" 200 10459 "-" "-"
[25/Mar/2019:15:37:46 +0000] "CONNECT registry.npmjs.org:443 HTTP/1.1" 200 10471 "-" "-"
[25/Mar/2019:15:37:46 +0000] "CONNECT registry.npmjs.org:443 HTTP/1.1" 200 9368 "-" "-"
/build/wekan/parts/wekan/install/lib/node_modules/api2html/bin/api2html.js:23
    "php: "PHP"
           ^^^

SyntaxError: Unexpected identifier
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
Failed to run 'override-build': Exit code was 1.
Build failed
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/build_snap.py", line 254, in run
    self.build()
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/build_snap.py", line 243, in build
    env=env)
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/build_snap.py", line 97, in run_build_command
    return self.backend.run(args, env=full_env, **kwargs)
  File "/usr/lib/python2.7/dist-packages/lpbuildd/target/lxd.py", line 497, in run
    subprocess.check_call(cmd, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '['lxc', 'exec', 'lp-xenial-amd64', '--env', 'LANG=C.UTF-8', '--env', 'SHELL=/bin/sh', '--env', 'SNAPCRAFT_BUILD_INFO=1', '--env', 'SNAPCRAFT_IMAGE_INFO={"build_url": "https://launchpad.net/~wekan-team/+snap/wekan-devel/+build/504568"}', '--env', 'SNAPCRAFT_BUILD_ENVIRONMENT=host', '--env', 'http_proxy=http://10.10.10.1:8222/', '--env', 'https_proxy=http://10.10.10.1:8222/', '--env', 'GIT_PROXY_COMMAND=/usr/local/bin/snap-git-proxy', '--', '/bin/sh', '-c', 'cd /build/wekan && linux64 snapcraft']' returned non-zero exit status 2
Revoking proxy token...
RUN: /usr/share/launchpad-buildd/bin/in-target scan-for-processes --backend=lxd --series=xenial --arch=amd64 SNAPBUILD-504568
Scanning for processes to kill in build SNAPBUILD-504568
```
@bentiss 

Do you have ideas how to fix this?

**Comment by bentiss** [comment]
Looks like this was added by https://github.com/tobilg/api2html/commit/a9a41bca18db3f9ec61395d7262eff071a995783 and included in 0.3.1 of `api2html`.

Can you try forcing 0.3.0 of `api2html` in snap.yml?

**Comment by xet7** [comment]
@bentiss 

Thanks for finding that, I'll try with specifying version.
